scopus-import/src/main/java/no/sikt/nva/scopus/conversion/files/ScopusFileConverter.java

```diff
@@ -290,12 +292,22 @@ private ScopusFile toScopusFile(CrossrefLink crossrefLink, List<License> license
         return ScopusFile.builder()
                    .withIdentifier(randomUUID())
                    .withDownloadFileUrl(crossrefLink.getUri())
-                   .withPublisherAuthority(VOR.equals(crossrefLink.getContentVersion()))
+                   .withPublisherVersion(determinePublisherVersion(crossrefLink))
                    .withLicense(extractLicenseForLink(crossrefLink, licenses))
                    .withEmbargo(calculateEmbargo(crossrefLink, licenses))
                    .build();
     }
 
+    private PublisherVersion determinePublisherVersion(CrossrefLink crossrefLink) {
```

Comment:
I had no idea what those enums meant when I started with this, but I also found some documentation: https://www.crossref.org/schema/crossref_output3.0.1.xsd